### DISCUSSION TITLE
Swivel - Flash To Video ( Big Sur Icon )

### DISCUSSION THREAD

**Discussion by sulflix** [elrumo/macOS_Big_Sur_icons_replacements#1492] [Icon request] [answered]
https://www.newgrounds.com/wiki/creator-resources/animation-resources/swivel-flash-to-video
https://files.catbox.moe/7dofdj.zip ( zip file with swivel icons up to 512x512 )

**Comment by vikasbhatia** [comment, marked as answer]
Your first link wasn't working, but [this is what I found](https://macosicons.com/#/u/sidewinder) when I googled Swivel, so I made the first icon based on those colours. I also added black and white options.

Here you go:
<img width="128" alt="Swivel-1" src="https://github.com/user-attachments/assets/570c76c4-851d-4e03-aa77-dbe6c07b2224">
<img width="128" alt="Swivel-2" src="https://github.com/user-attachments/assets/9e2778e4-4cda-4684-a335-3efa422d7684">
<img width="128" alt="Swivel-3" src="https://github.com/user-attachments/assets/cdf64121-1a5d-4241-8821-7b41b753e646">

You can download the PNGs here to start using already, and the ICNS files from [my profile](https://macosicons.com/#/u/sidewinder) later when they are approved.

Out of curiosity, what are you working on in 2024 that requires Flash/a Flash converter?

**Reply by sulflix** [reply]
Thank you very much for making the icons, also, to answer your question, I animate using Adobe Animate (formerly Adobe Flash but i like to call it Flash) and using Adobe Media Encoder to export my animations makes it low quality so i use Swivel instead.

**Reply by vikasbhatia** [reply]
Oh I had no idea that Flash lived on as Adobe Animate! Thanks for educating me, you learn something new every day haha.

PS. in case you missed it, I also did your other Firefox Nightly request. Good luck with your projects.
